source/tyr/tests/integration/users_test.py

```diff
@@ -655,13 +589,16 @@ def test_get_users(create_multiple_users):
     """
     resp = api_get('/v0/users')
 
-    assert resp[0]['has_shape'] == True
-    assert resp[0]['shape'] == {}
-    assert resp[1]['has_shape'] == False
-    assert resp[1]['shape'] == None
+    for user in resp:
```

Comment:
It's completly minor, but I prefer to write this:

```python
foo = next((u for u in resp if u.get('login') == 'foo'), None)
assert foo
assert foo.get('has_shape') is True
...

```
and same for the other user (you should get his name)

I think it's more robust and the test intent is clearer